src/systemctl/systemctl.c

```diff
@@ -716,7 +720,18 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
                         break;
 
                 case 'i':
-                        arg_ignore_inhibitors = true;
+                        arg_check_inhibitors = -1;
+                        break;
+
+                case ARG_CHECK_INHIBITORS:
+                        if (streq(optarg, "auto"))
+                                arg_check_inhibitors = 0;
+                        else {
+                                r = parse_boolean(optarg);
+                                if (r < 0)
+                                        return log_error_errno(r, "Failed to parse --check-inhibitors= argument: %s", optarg);
+                                arg_check_inhibitors = r;
```

Comment:
hmm? `--ignore-inhibitors` and `--check-inhibitors=auto` should be synonymous. i.e. arg_check_inhibitors should be set to -1 in the "auto" branch here too